spec/requests/api/ants/available_time_slots_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+describe "ANTS API: availableTimeSlots" do
+  around do |example|
+    previous_auth_token = ENV["ANTS_API_AUTH_TOKEN"]
+
+    ENV["ANTS_API_AUTH_TOKEN"] = "fake_ants_api_auth_token"
+
+    example.run
+
+    ENV["ANTS_API_AUTH_TOKEN"] = previous_auth_token
+  end
```

Comment:
maintenant qu'on a trois fichiers différents qui utilisent ce block `around`, ça pourrait être bien de le mettre en commun via un module ou autre chose

Reply:
Done in a16f94302b3e374f015aa94a1a25a5eeb3a9d874

Reply:
nickel